src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/dailyquest/QuestLoader.kt

```diff
@@ -36,12 +35,12 @@ class QuestLoader(private val dailyQuestHelper: DailyQuestHelper) {
         }
     }
 
-    fun loadFromTabList() {
+    fun loadFromTabList(lines: List<String>) {
```

Comment:
Why do we request the lines as parameter? they are always the same static data, TabWidget.FACTION_QUESTS.lines